homeassistant/bootstrap.py

```diff
@@ -207,6 +208,8 @@ def process_ha_core_config(hass, config):
         if key in config:
             setattr(hass.config, attr, config[key])
 
+    Entity.visibility.update(config.get('visibility', [{}])[0])
```

Comment:
To be honest, I didn't even think that was an option so I didn't even try it. I'll update this one as well. Should be another quick fix.